src/order/filter/archimedean.lean

```diff
@@ -135,3 +140,106 @@ begin
   ... = (max b 0 * n) * r : by rw [mul_assoc]
   ... ≤ f x * r : mul_le_mul_of_nonneg_right hx (le_of_lt hr)
 end
+
+end linear_ordered_semiring
+
+section linear_ordered_ring
+
+variables [linear_ordered_ring R] [archimedean R]
+
+/-- See also `filter.tendsto.at_top_mul_neg_const` for a version of this lemma for
+`linear_ordered_field`s which does not require the `archimedean` assumption. -/
+lemma tendsto.at_top_mul_neg_const' (hr : r < 0) (hf : tendsto f l at_top) :
```

Comment:
Shouldn't the name mention `at_bot`, because that's what the goal is about?

Reply:
I think you're right BUT I am following the naming of the existant lemma [https://leanprover-community.github.io/mathlib_docs/order/filter/at_top_bot.html#filter.tendsto.at_top_mul_neg_const](filter.tendsto.at_top_mul_neg_const) in the same way that the lemma `filter.tendsto.at_top_mul_const'` (a few lines up) follows [https://leanprover-community.github.io/mathlib_docs/order/filter/at_top_bot.html#filter.tendsto.at_top_mul_const](filter.tendsto.at_top_mul_const).

I'd be in favour of fixing these names. I'm very happy to do it as part of this PR if you would like, but it might be easier to separate that work out into a "rename-only" PR.